### DISCUSSION TITLE
What do to if you've partially moved your colony but the land you're not using won't unclaim?

### DISCUSSION THREAD

**Discussion by wastednumbers** [ldtteam/minecolonies#9859] [Q&A] [answered]
Hi,

So i started a colony and midway through I changed my style from the default to Caledonia. The box in the upper right is my old uni spot that I deconstructed and removed the hut block from. I also removed my guard towers and other huts from the the surrounding chunks. Now I'm expanding to the SW but I'm not sure if I'm going to run into problems later so I figure i'd ask now. Any insight is appreciated. I put a second smaller box for where the townhall is. This is a screenshot from JourneyMap

![image](https://github.com/ldtteam/minecolonies/assets/99608230/e7e902ef-3d7b-4920-8fd2-6e8362f9e94c)

Version i'm using:
![image](https://github.com/ldtteam/minecolonies/assets/99608230/926efad3-2575-41e9-9447-7f43305eb207)

and this is the curseforge mod pack if you wanna see more details
![image](https://github.com/ldtteam/minecolonies/assets/99608230/3a367981-c4d4-473b-98eb-07116f005654)

**Comment by Raycoms** [comment, marked as answer]
Don't worry, the claim limit is a distance thing not a "max number of chunks". The old colony center stays where it was and stays claimed. You'd have to delete the colony entirely to actually move that.

**Reply by wastednumbers** [reply]
Thank you! i have some underground stuff that i was worried i was going to accidently build over, hence the move :)
